client/src/pages/JoinGame.tsx

```diff
@@ -11,6 +13,15 @@ const JoinGame = () => {
     socket.emit("JOIN_GAME", gameCode, playerName);
   };
 
+  const [gameCodeCreated, setGameCode] = useState("");
```

Comment:
Whats the difference between the two game code states. Also we need to name the setter function as the same as the state with "set" prefixed.